module0/prework/wsl-computer-setup.md

```diff
@@ -0,0 +1,112 @@
+    <img src="./assets/wsl/WSL-6.png" alt="" style="width: 80%">
+
+----------------------------------------------------
+
+## Install Ruby
+_All commands will be run in your Ubuntu Terminal_
+1. update the apt package manager with `sudo apt-get update`
+    * If asked for your password, enter the one you configured during the WSL setup above.
+2. install ruby with `sudo apt-get install ruby-full`
+    * enter 'Y' to continue through all prompts
+3. Confirm that ruby has been installed by opening irb (interactive ruby)
+    * `irb`
+    * you can play around with ruby code here!
+    * when you are ready to exit irb, enter `exit!`  
+    <img src="./assets/wsl/WSL-8.png" alt="" style="width: 60%">
+4. Configure your gem settings (don't worry about what gems are right now)
+    * `echo '# Install Ruby Gems to ~/gems' >> ~/.bashrc`
+    * `echo 'export GEM_HOME="$HOME/gems"' >> ~/.bashrc`
+    * `echo 'export PATH="$HOME/gems/bin:$PATH"' >> ~/.bashrc`
+    * `source ~/.bashrc`  
+    <img src="./assets/wsl/WSL-9.png" alt="" style="width: 80%">
+5. Confirm that your gem setup is configured correctly
+    * `gem install pry`
+    * `pry` - this opens an interactive ruby environment similar to irb
+    * `exit!`  
+    <img src="./assets/wsl/WSL-10.png" alt="" style="width: 80%">
+
+---------------------------------------------------------------------
+
+## Install VSCode
+_VSCode is the text editor that we will use for writing code_
+1. Download and Install VSCode
+    * Use the ['Windows Installer'](https://code.visualstudio.com/docs/setup/windows)
+    * Keep all default settings in the install wizard
+2. Close and Restart PowerShell
+    * When you restart, you will need to switch to Ubuntu  
+    <img src="./assets/wsl/WSL-11.png" alt="" style="width: 80%">
+3. Verify setup
+    * create a new file with `touch ruby-playground.rb`
+    * open the file in VSCode with `code ruby-playground.rb`
+    <img src="./assets/wsl/WSL-12.png" alt="" style="width: 80%">
+    * Add the following line to the file and save the file (ctrl+s): `puts "Hello World"`  
+    <img src="./assets/wsl/WSL-13.png" alt="" style="width: 80%">
+    * In PowerShell, run the file with `ruby ruby-playground.rb`  
+    <img src="./assets/wsl/WSL-14.png" alt="" style="width: 80%">
+
+**Congrats! You have created and run your first Ruby program!!**
+
+------------------------------------------------------------------------
+
+## Configure GitHub
+1. [Create an account on GitHub](https://github.com/)
+2. Configure your local git settings
+    * Run the following commands in PowerShell (make sure you are in your WSL Ubuntu environment!) and substitute your info for the first two settings:
+    * `git config --global user.name "Megan McMahon"`
+    * `git config --global user.email megan@example.com`
+    * `git config --global init.defaultBranch main`
+    * `git config --global core.editor "code --wait"`
+    * `git config --global pull.rebase false`
+3. Configure SSH
```

Comment:
Same comment as above - consolidate this in 1 place in the repo. And GitHub now recommends `ed25519` keys instead of rsa keys. https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent

Reply:
Ooh! Thank you!  I will look into this for an update to both setup instructions!!